server/sonar-server/src/main/java/org/sonar/server/issue/InternalRubyIssueService.java

```diff
@@ -314,6 +316,20 @@ private static Date checkAndReturnDeadline(String deadLineParam, Result<ActionPl
     return result;
   }
 
+  public Result<Issue> executeAction(String issueKey, String actionKey) {
+    Result<Issue> result = Result.of();
+    try {
+      result.set(actionService.execute(issueKey, actionKey));
+    } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)